### PR TITLE
Fix Code.Sample and Config.Sample

### LINKED ISSUE #385: Session affinity scenario is broken in the sample app
Session affinity doesn't work in ReverseProxy.Config.Sample. It might have got broken when transformation pieces were added.

### PR DESCRIPTION
**Pull request by alnikola**
Fixes reverse proxy samples:
- Removes redundant `HttpController` from Code.Sample and Config.Sample which is preventing the proxy pipeline from being invoked
- Moves a path transform example to 'cluster2' section in order to fix 'cluster1' path in Config.Sample

Fixes #385

### CODE REVIEW

File: docs/docfx/articles/proxyhttpclientconfig.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ public sealed class ProxyHttpClientOptions
 ```
 Note that instead of defining certificate location as it was in [CertificateConfigData](xref:Microsoft.ReverseProxy.Configuration.Contract.CertificateConfigData) model, this type exposes a fully constructed [X509Certificate](xref:System.Security.Cryptography.X509Certificates.X509Certificate) certificate. Conversion from the configuration contract to the abstraction model is done by a [IProxyConfigProvider](xref:Microsoft.ReverseProxy.Service.IProxyConfigProvider) which loads a client certificate into memory.
 
-The following is an example of `ProxyHttpClientOptions` using [code based](xref:configproviders) configuration. An instance of `ProxyHttpClientOptions` is assigned to the [Cluster.HttpClientOptions](xref:Microsoft.ReverseProxy.Abstractions.Cluster.HttpClientOptions) property before passing the `Cluster` array to `LoadFromMemory` method.
+The following is an example of `ProxyHttpClientOptions` using [code based](configproviders.md) configuration. An instance of `ProxyHttpClientOptions` is assigned to the [Cluster.HttpClientOptions](xref:Microsoft.ReverseProxy.Abstractions.Cluster.HttpClientOptions) property before passing the `Cluster` array to `LoadFromMemory` method.
```
Comment (Tratcher) [inline docs/docfx/articles/proxyhttpclientconfig.md:135]: Did you verify this works in the build output by running `dotnet serve`? The docfx page links aren't quite the same as markdown links. I copied this format from https://github.com/microsoft/reverse-proxy/blame/release/docs/docs/docfx/index.md#L16 which is working in the rendered docs: https://microsoft.github.io/reverse-proxy/

Comment (alnikola) [reply docs/docfx/articles/proxyhttpclientconfig.md:135]: `[code based](xref:configproviders) ` was producing a warning on local build. Once I changed it to `[code based](configproviders.md)` it's been gone.

Comment (alnikola) [reply docs/docfx/articles/proxyhttpclientconfig.md:135]: Just confirmed the new version works correctly with `dotnet-serve`.

Comment (Tratcher) [reply docs/docfx/articles/proxyhttpclientconfig.md:135]: Hmm, and the link in index.md doesn't produce the same warning?

**Review by Tratcher — APPROVED** [review]

File: docs/docfx/articles/proxyhttpclientconfig.md (hunk 1 above)
Comment (alnikola) [reply docs/docfx/articles/proxyhttpclientconfig.md:135]: No, surprisingly. I don't know what is the difference here.

**Pull request merged**